core/src/main/java/com/redhat/lightblue/client/request/data/DataFindRequest.java

```diff
@@ -35,11 +34,23 @@ public void where(Query queryExpression) {
     }
 
     public void select(List<? extends Projection> projections) {
+        select(projections, null, null);
+    }
+
+    public void select(List<? extends Projection> projections, Integer begin, Integer end) {
         projection = Projection.project(projections);
+        this.begin = begin;
+        this.end = end;
     }
 
     public void select(Projection... projection) {
+        select(projection, null, null);
+    }
+
+    public void select(Projection[] projection, Integer begin, Integer end) {
```

Comment:
This does sour syntactical sugar somewhat. For this variation you would need to declare the array. I just don't like how (int, int, Projection...) read, but I am open to opposition on this point. This is backwards compatible as the method variation is net new.

Example:
```java
select(new Projection[]{Projection1, Projection2}, int, int);
```